### PR TITLE
Fixed CoD stats, and made the output stat between 0 - 1

### PR DESCRIPTION
**Pull request by kywch**

### CODE REVIEW

**Review by daveey — COMMENTED** [review]
this will log stats on every tick, can we just log at end of episode?

**Review by daveey — COMMENTED** [review]
These have to be cleared in reset(), also you're logging on agent_death, i think we only want to log on end of episode.

File: nmmo_env.py [hunk 1]
```diff
@@ -92,38 +94,69 @@ def __init__(self, args: Namespace):
...
-  #   super().__init__(env, teams, team_id)
-  # self._feature_extractor = FeatureExtractor(teams, team_id, env.config)
+  def __init__(self, env, teams, team_id):
+    super().__init__(env, teams, team_id)
 
-  # def reset(self, team_obs):
-  #   super().reset(team_obs)
-  # self._feature_extractor.reset(team_obs)
+    # stat vars
+    self._cod_attacked = 0
```
Comment (daveey) [inline nmmo_env.py]: how about moving these into _reset_episode_stats() so that we only have one place to modify

File: nmmo_env.py [hunk 2]
```diff
@@ -92,38 +94,69 @@ def __init__(self, args: Namespace):
...
+      team_infos["stats"]["cod/starved"] = self._cod_starved
+      team_infos["stats"]["cod/dehydrated"] = self._cod_dehydrated
+      team_infos["stats"]["task/completed"] = self._task_completed
+
+      achieved, performed, event_cnt = \
+        process_event_log(self.env.realm, self.teams[self.team_id])
+      for key, val in list(achieved.items()) + list(performed.items()):
+        team_infos["stats"][key] = float(val)
+
+      print()
```
Comment (daveey) [inline nmmo_env.py]: remove

**Review by daveey — APPROVED** [review]

File: nmmo_env.py (hunk 1 above)
Comment (kywch) [reply nmmo_env.py]: added _reset_episode_stats()

File: nmmo_env.py (hunk 2 above)
Comment (kywch) [reply nmmo_env.py]: oh shoot, removed!

**Comment by kywch** [comment]
Also moved `pre-git-check.sh` to the `scripts` folder. 

Merging to the 2.0 branch, thanks!

**Pull request merged**